Meshtastic/Views/Layouts/TraceRoute.swift

```diff
@@ -56,13 +56,13 @@ struct TraceRoute: Layout {
 
 			subview.place(at: point, anchor: .center, proposal: .unspecified)
 
-			DispatchQueue.main.async {
+		//	DispatchQueue.main.async {
```

Comment:
I was running this branch earlier today and noticed that this change was causing the arrows between nodes to no longer angle correctly. They all pointed one direction until I uncommented these.

Reply:
For me, the app crashed when going from a multi-hop traceroute to a direct traceroute in the list. With this, this doesn't happen anymore, but indeed the arrows don't angle correctly anymore.

Reply:
Ok. I reproduced that crash. Looks like it's probably referencing the subview data after it's getting torn down. 
On the flip side, those assignments in this state trigger a "Modifying state during view update, this will cause undefined behavior” warning.